domains/cabbage.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "Aaron's personal developer website",
+    "repo": "https://github.com/aaron-harvey",
+    "owner": {
+        "username": "cabbage",
+        "email": "1255651384@qq.com"
```

This doesn't seem like your real email address. We need a way to contact you in case you need to be informed of some change.

If you don't wish to share your email with us, you can share your twitter/mastodon handle or any other social media username.
```suggestion
        "email": "",
        "twitter": "yourusername",
```